test/Microsoft.AspNetCore.Server.KestrelTests/HttpsConnectionFilterTests.cs

```diff
@@ -330,6 +331,32 @@ public void ThrowsWhenNoServerCertificateIsProvided()
             }
         }
 
+        [Fact]
+        public async Task ClientHandshakeFailureLoggedAsInformation()
+        {
+            var logger = new HandshakerErrorLogger();
+
+            var serviceContext = new TestServiceContext(new HttpsConnectionFilter(
+                new HttpsConnectionFilterOptions
+                { ServerCertificate = _x509Certificate2 },
```

Comment:
nit: put in the same line as the `new` or make a multiline block, this reads a bit weird.